### PR TITLE
make notes more easily to understand

### PR DESCRIPTION
**Pull request by mamil**
#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/internal/queue/scheduling_queue.go [hunk 1]
```diff
@@ -529,7 +529,7 @@ func (p *PriorityQueue) AssignedPodUpdated(pod *v1.Pod) {
 
 // MoveAllToActiveOrBackoffQueue moves all pods from unschedulableQ to activeQ or backoffQ.
 // This function adds all pods and then signals the condition variable to ensure that
-// if Pop() is waiting for an item, it receives it after all the pods are in the
+// if Pop() is waiting for an item, it receives signals after all the pods are in the
```
Comment (Huang-Wei) [inline pkg/scheduler/internal/queue/scheduling_queue.go:532]:
```suggestion
// if Pop() is waiting for an item, it receives the signal after all the pods are in the
```

Comment (mamil) [reply pkg/scheduler/internal/queue/scheduling_queue.go]: done! thanks for your comment~

**Comment by dims** [comment]
@Huang-Wei needs an ok-to-test first ;)

/ok-to-test

**Comment by Huang-Wei** [comment]
> @Huang-Wei needs an ok-to-test first ;)

Aha, missed that... Thanks.

**Pull request merged**